nornir/plugins/functions/text/__init__.py

```diff
@@ -41,7 +44,15 @@ def _print_individual_result(
     failed: bool,
     severity_level: int,
     task_group: bool = False,
-) -> None:
+    return_output: bool = False,
```

Comment:
same as below, instead of using a bool just pass here the destination where you want to write the `print`. Difference here is that the parameter should be mandatory.